operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [CHANGE] Use custom memcached templates to break bitnami dependency. #2064
```

Comment:
We should probably note that this increases the default resource usage a bit since memcached is enabled by default now.

Reply:
This was completely unintentional. 

Memcached components should be disabled by default, the same way they are now. Gonna change it.